source/NeutronaWand/NeutronaWand.ino

```diff
@@ -1869,7 +1869,7 @@ void mainLoop() {
   }
 
   if(ms_firing_lights_end.justFinished()) {
-    fireStreamEnd(getHueAsGRB(C_BLACK));
+    fireStreamEnd(getHueColour(C_BLACK, i_num_barrel_leds));
```

Comment:
Related to the request in Colours.h, if we could use an ENUM to track the type of LED configuration vs. the LED count, that may be easier for maintenance (given that this designation covers both a count of LED's as well as the RGB/GRB status).